spec/support/app_config_helpers.rb

```diff
@@ -0,0 +1,7 @@
+module AppConfigHelpers
+  def mock_app_config(**attrs)
+    ac = spy('AppConfig', require_phone: false, **attrs)
+    allow(AppConfig).to receive(:first) { ac }
```

Comment:
This is great! I'd just reiterate my style preference from [before](https://github.com/YaleSTC/reservations/pull/1566/files#r63 642716), but it's so nice to start using proper testing methods :smile:.